tests/py-flux-test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import numpy as np
+import pandas as pd
```

Comment:
Would be nice to avoid this dependency. It shouldn't be too hard to replace this with Python's csv package (in the standard library).

Reply:
Not even numpy? Lists are fine, but I'm not even sure `PccThermalModel1D` would accept lists as input (e.g., for z)

Reply:
Oh, oops... I just meant pandas. ;-) I don't use it much and would prefer to just skip to numpy if it's not too much of an inconvenience.

Reply:
that's ok and I noticed that you were still referring to np below (and thus I deleted my comment). pd are quite cool and very flexible in reading ascii files, but they can be heavy and unnecessary here, indeed. ;)